Makefile

```diff
@@ -141,7 +141,7 @@ lint:
 		--enable unconvert \
 		--enable varcheck
 
-	golangci-lint run --tests false --disable-all \
+	golangci-lint run --tests=false --disable-all \
```

Comment:
this change seems unnecessary?

Reply:
For me, withouth --tests=false it is running also for the tests. Will try to verify this again.

Reply:
Just removed the =. Now i am getting this:

`WARN [runner/unused] Can't run megacheck because of compilation errors in packages [github.com/hairyhenderson/gomplate/false]: -: cannot find package "github.com/hairyhenderson/gomplate/false" in any of:
        /usr/local/Cellar/go/1.12.5/libexec/src/github.com/hairyhenderson/gomplate/false (from $GOROOT)
        /Users/nexeck/go/src/github.com/hairyhenderson/gomplate/false (from $GOPATH)
`

It looks like he is trying to find a package named false without the =

Reply:
ah. I thought this was working - for some reason the CI passes even though the syntax is wrong: https://circleci.com/gh/hairyhenderson/gomplate/2225

Carry on! 😂